crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -150,6 +150,36 @@ impl ApiServer {
             EthRequest::EvmMine(mine) => self.evm_mine(mine).await.to_rpc_result(),
             EthRequest::EvmMineDetailed(mine) => self.evm_mine_detailed(mine).await.to_rpc_result(),
 
+            // ---- Coinbase ----
+            EthRequest::SetCoinbase(address) => {
+                node_info!("anvil_setCoinbase");
+                let latest_block = self.latest_block();
+                let account_id = self
+                    .client
+                    .runtime_api()
+                    .account_id(latest_block, H160::from_slice(address.as_slice()))
+                    .map_err(|err| Error::InvalidParams(format!("{err}")));
```

Comment:
the main `Error` enum already wraps the `sp_api::ApiError` error. I don't think it's worth being this verbose with the error messages for now, we can just use the `?` operator. Same for the runtime API call in EthCoinbase